File: BackEnd/IBMYoung/Model/Questao.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace IBMYoung.Model
{
    public class Questao
    {
        public int Ordem { get; set; }

        public string Titulo { get; set; }

        public string Conteudo { get; set; }

        [ForeignKey(nameof(TarefaId))]
        public virtual Tarefa Tarefa { get; set; }

        public int TarefaId { get; set; }

        public virtual List<Alternativa> Alternativas { get; set; }
    }
}

File: Mobile/IBM_Young/app/src/main/java/br/senai/sp/informatica/ibmyoung/model/Questao.java
package br.senai.sp.informatica.ibmyoung.model;

import java.security.PrivateKey;
import java.util.List;

/**
 * Created by pena on 31/03/2018.
 */

public class Questao {
    private int ordem;
    private String titulo;
    private String conteudo;
    private int tarefaId;
    private boolean respondida;
    private List<Alternativa> alternativas;

    public int getOrdem() {
        return ordem;
    }

    public void setOrdem(int ordem) {
        this.ordem = ordem;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getConteudo() {
        return conteudo;
    }

    public void setConteudo(String conteudo) {
        this.conteudo = conteudo;
    }

    public int getTarefaId() {
        return tarefaId;
    }

    public void setTarefaId(int tarefaId) {
        this.tarefaId = tarefaId;
    }

    public boolean isRespondida() {
        return respondida;
    }

    public void setRespondida(boolean respondida) {
        this.respondida = respondida;
    }

    public List<Alternativa> getAlternativas() {
        return alternativas;
    }

    public void setAlternativas(List<Alternativa> alternativas) {
        this.alternativas = alternativas;
    }
}
